sale_order_type/models/sale_order.py

```diff
@@ -36,6 +36,23 @@ def onchange_type_id(self):
             if order.type_id.incoterm_id:
                 order.incoterm = order.type_id.incoterm_id.id
 
+    @api.multi
+    @api.onchange('order_line')
+    def _onchange_order_line(self):
```

Comment:
This onchange shouldn't be triggered automatically.

Reply:
Hi, thanks for the review!
It is triggered automatically but it only does something if the `type_id` of the order is empty.
Are you suggesting it should depend on something else or should I remove it completely?

Reply:
Done in https://github.com/OCA/sale-workflow/pull/770/commits/65a7dc240c43f0426b00431a48e3c75e8361c0e1